CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ If you must make some internal code `public` to access it from other packages, p
 
 ### Forward
 
-Completely new API should, in most cases, be marked with `@Beta`. This lets users know not to rely too much on your changes yet, as the public release might reveal that more work needs to be done.
+Completely new APIs should be marked with `@Alpha` or `@Beta`. This lets users know not to rely too much on your changes yet, as the public release might reveal that more work needs to be done.
```

Comment:
We could add hyperlinks to Javadoc of the Alpha / Beta annotations. The Javadoc distinguishes them nicely.

Reply:
good idea.